content/open-source/octo-sts/overview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Octo STS"
+linktitle: "Octo STS"
+type: "article"
+lead: "A security token service that eliminates the need for GitHub Personal Access Tokens by enabling OIDC-based federation for GitHub API access"
+description: "Learn about Octo STS, an open source security token service for GitHub that uses OIDC federation to eliminate long-lived Personal Access Tokens"
+date: 2025-12-23T15:04:05+01:00
+lastmod: 2025-12-23T15:04:05+01:00
+tags: ["octo-sts", "Overview", "OIDC", "Security"]
+draft: false
+images: []
+menu:
+  docs:
+    parent: "open-source"
+weight: 100
+toc: true
+---
+
+Octo STS is a GitHub App developed by Chainguard that acts as a Security Token Service (STS) for the GitHub API. It enables workloads running anywhere that can produce OIDC tokens to federate with GitHub, exchanging those tokens for short-lived GitHub access tokens. The primary goal is to eliminate the need for GitHub Personal Access Tokens (PATs), which are long-lived credentials that pose significant security risks.
+
+## Why Octo STS Matters
+
+Long-lived access tokens are a common target in security incidents. When attackers gain access to a PAT, they can exploit it to access repositories, make changes, and pivot to other resources. These tokens often have broad permissions and no expiration date, making them particularly dangerous if compromised.
+
+Octo STS addresses this problem by:
+
+- **Eliminating long-lived credentials**: No more PATs that sit around indefinitely
+- **Enabling OIDC federation**: Leverage existing identity providers to authenticate workloads
+- **Providing short-lived tokens**: Generated tokens expire automatically
+- **Implementing fine-grained access control**: Grant only the permissions needed for specific tasks
+- **Supporting multiple identity providers**: Works with GitHub Actions, cloud providers (AWS, GCP, Azure), Kubernetes, and any OIDC-compliant system
+
+## How Octo STS Works
+
+Octo STS operates through a trust policy model:
+
+1. **Install the GitHub App**: Add the [octo-sts](https://github.com/apps/octo-sts) GitHub App to your organization or repositories
+2. **Define trust policies**: Create policy files that specify which identities can access which resources
+3. **Exchange tokens**: Workloads present OIDC tokens to Octo STS
+4. **Receive GitHub tokens**: If the identity matches the trust policy, Octo STS issues a short-lived GitHub token with specified permissions
+
+### The Token Exchange Process
+
+```
+[Workload] → OIDC Token → [Octo STS] → Validates → Trust Policy
+                                          ↓
+                            Short-lived GitHub Token ← Returns
+```
+
+The workload can then use this GitHub token to interact with the GitHub API, open pull requests, push commits, or perform other operations as defined by the trust policy.
+
+## Key Features
```

Comment:
I don't like the way this renders. I think you either need another line between the bold portion and the explanation, or turn these into an unordered list or table or something. Right now these look funky